functional_tests/scylla_operator/test_functional.py

```diff
@@ -332,3 +332,21 @@ def test_scylla_operator_pods(db_cluster: ScyllaPodCluster):
 
     not_running_pods = ','.join([pods_info[0] for pods_info in scylla_operator_pods if pods_info[1] != 'Running'])
     assert not not_running_pods, f'There are pods in state other than running: {not_running_pods}'
+
+
+def test_startup_probe_in_scylla_pods(db_cluster: ScyllaPodCluster):
+    pods = db_cluster.k8s_cluster.kubectl(f'get pods -A -l "scylla/cluster=sct-cluster" -o yaml')
+
+    no_startup_prob_pods = []
+    for pod in yaml.load(pods.stdout)["items"]:
+        if "spec" not in pod:
+            continue
+
+        if "containers" not in pod["spec"]:
+            continue
+
+        for container in pod["spec"]["containers"]:
+            if container['name'] == "scylla" and "startupProbe" not in container:
```

Comment:
What if "startupProbe" is empty itself? We should also check that it is populated.

Reply:
added validation for non-empty startupProbe value